### ISSUE TITLE
Problems on Windows with umlauts ä ö ü in path name to g code file

### ISSUE THREAD

**Issue by ModuloFS** [CauldronDevelopmentLLC/CAMotics#386]
Hello,

if there is a umlaut like ä, ü, or ö in any position of the path to the stored g-code file. The simulation of the path is not loaded / working. A Error message is not shown, because of that it takes some time to find the failure for a new user. 

Anybody else has the same problem?

**Comment by gadelan** [comment]
Yes. It also fails when using other extended latin-1 characters like á é ç in the .camotics project file path. Also Windows.
